packages/status-js/src/client/community/community.ts

```diff
@@ -85,380 +66,125 @@ export class Community {
         while (--index >= 0) {
           this.client.handleWakuMessage(wakuMessages[index])
 
-          if (!this.communityMetadata) {
+          if (!this.description) {
             return shouldStop
           }
 
-          communityMetadata = this.communityMetadata
+          description = this.description
           shouldStop = true
 
           return shouldStop
```

Comment:
Can we just have this condition instead of this block of code:
```js
return this.description !== undefined
```

Reply:
We're also returning the `description`. Do you think the function could be also called from elsewhere other than `start()`.

If not, then yes I can see doing that and accessing it via `this.description` in `start()` where we check it's set.